testsuite/Rakefile

```
@@ -129,4 +146,111 @@ namespace :utils do
     sh 'find . -size 0 -delete', verbose: false
     sh 'node index.js &> cucumber_reporter.log', verbose: false
   end
+
+end
+
+namespace :jenkins do
+
+  desc 'Generate rake file runstep'
+  task :generate_rake_run_step, [:extension, :client_name, :output_path, :yaml_path] do |t, args|
+    output_filepath = "#{args[:output_path]}/#{args[:client_name]}#{args[:extension]}.feature"
+    sh "echo '- #{output_filepath}' >> #{args[:yaml_path]}", verbose: false
+  end
+
+  desc 'Generate rake files for create bootstrap repositories'
+  task :generate_rake_files_create_bootstrap_repositories, [:instances] do |_t, args|
+    minions = args[:instances]
+    minions.each do |minion|
+      if minion.include?('ssh')
+        next
+      else
+        Rake::Task['jenkins:generate_rake_run_step'].invoke("_create_bootstrap_repository",
+                                                            File.basename(minion,".feature"),
+                                                            "features/build_validation/create_bootstrap_repositories",
+                                                            "run_sets/build_validation_create_bootstrap_repository_#{minion}.yml")
+        Rake::Task['jenkins:generate_rake_run_step'].reenable
+      end
```

```suggestion
      next if minion.include?('ssh')

      Rake::Task['jenkins:generate_rake_run_step'].invoke("_create_bootstrap_repository",
                                                          File.basename(minion,".feature"),
                                                          "features/build_validation/create_bootstrap_repositories",
                                                          "run_sets/build_validation_create_bootstrap_repository_#{minion}.yml")
      Rake::Task['jenkins:generate_rake_run_step'].reenable
```
This could be refactored in a guard clause way instead of wrapping the code inside the if conditional. Same for the occurrences below. But feel free to ignore it because I did not test this, however it should work :wink: